ProjectCoimbra.UWP/Project.Coimbra/Project.Coimbra.csproj

```diff
@@ -35,10 +35,11 @@
     <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>

```

Comment:
Are the changes in this file deliberate?

Reply:
Woops these changes got merged by mistake, I will revert.